configure.ac

```diff
@@ -61,8 +61,8 @@ AC_PROG_RANLIB
 AC_HEADER_STDC
 AC_CHECK_HEADERS([infiniband/verbs.h],,[AC_MSG_ERROR([ibverbs header files not found])])
 AC_CHECK_LIB([ibverbs], [ibv_get_device_list], [], [AC_MSG_ERROR([libibverbs not found])])
-AC_CHECK_LIB([rdmacm], [rdma_create_event_channel], [], AC_MSG_ERROR([librdmacm-devel not found]))
-AC_CHECK_LIB([ibumad], [umad_init], [LIBUMAD=-libumad], AC_MSG_ERROR([libibumad not found]))
+AC_CHECK_LIB([rdmacm], [rdma_create_event_channel], [], AC_MSG_ERROR([librdmacm-dev not found]))
```

Comment:
this change is only mentioning the debian packages which is not correct.
maybe the best way to do is mentioning both (debian and rpm).

for example here:
rdmacm wasn't found, consider installing librdmacm-dev or librdmacm-devel. (this is mentioning both rpm and debian naming)

Reply:
good point. fine with only updating error messages for including both rpm and deb? or something else?